hack/e2e.sh

```diff
@@ -109,6 +109,12 @@ go test \
     ${EXTRA_TEST_FLAGS} \
     2>&1 | tee "$TEST_LOG_FILE_PATH" ; TEST_EXIT_CODE=${PIPESTATUS[0]}
 
+go test \
```

Comment:
oh I get it, 2nd call to `go test`. But what if we made the one call with something like:
```go test -run='Test*' ./test/...```

Reply:
It would fail to run anything successfully as we're providing custom go test flags and there's no logic to consume those flags (besides the e2e package) which results in an error.

Reply:
✅ 
We should do the package dir reorg you mention in the init comment at some point then.

Reply:
Yeah completely agree. I'm not a huge fan of this approach, but it at least gets us back up and running until we have some dedicated time to re-organize these testing packages.